presto-parser/src/test/java/com/facebook/presto/sql/parser/TestStatementBuilder.java

```diff
@@ -95,6 +95,11 @@ public void testStatementBuilder()
                 "from emp");
 
         printStatement("" +
+                "select depname, empno, salary\n" +
+                ", lag(salary, 1) ignore nulls over (partition by depname)\n" +
```

Comment:
Move this line to the above `printStatement` call that has all the other window functions. No need to introduce a new query. Also add a test for `respect nulls`.